README.md

```diff
@@ -35,21 +35,22 @@ Go to the [ng-jhipster project](https://github.com/jhipster/ng-jhipster) and cli
 
 ### Build
 
-Run `yarn install` to install all dependencies.
+Run `npm install` to install all dependencies.
 
-Make some changes, run `yarn run test` to run both tslint and karma tests.
+Make some changes, run `npm run test` to run both tslint and karma tests.
 
-Build the library with `yarn run build` and then `yarn ngc`.
+Build the library with `npm run build` and then `npm run ngc`.
 
-Package the library with `yarn pack`, this will create an archive `ng-jhipster-vX.Y.Z.tgz`.
+Package the library with `npm run build`.
 
-For testing, you will want to integrate this archive into an application generated by JHipster.
+### Set NPM to use the cloned project
 
-Go to your generated JHipster application and run...
+In your cloned `ng-jhipster` project, type `npm link`.
 
-    git checkout -- yarn.lock
-    yarn cache clean ng-jhipster
-    rm -rf `yarn cache dir`/.tmp
-    yarn add path/to/ng-jhipster/ng-jhipster-vX.Y.Z.tgz
+This will do a symbolic link from the global `node_modules` version to point to this folder.
```

Comment:
Did you check this does it work? I guess I was using the pack command with NPM as well since the linking part was not working with AOT

Reply:
I use the same README than React, but to be honest, I didn't test this